src/UI/MainMenuOptions.cpp

```diff
@@ -70,6 +70,11 @@ void MainMenuOptions::init()
     cmbResolution.maxDisplayItems(1);
 
     cmbResolution.clearSelection();
+    { //Default resolution
+        std::ostringstream ss;
+        ss << constants::DEFAULT_WINDOW_WIDTH << 'x' << constants::DEFAULT_WINDOW_HEIGHT << 'x' << 60;
+        cmbResolution.text(ss.str());
+    }
```

Comment:
This part looks like it comes from another branch (#159). I think it may have been included here by accident.

Reply:
Notice #159 has the old name. This one is the new name. Because #159 is a prerequisite and they were created independently of each other, a merge conflict will need to be resolved.

Reply:
Notice #159 has the old name. This one is the new name. Because #159 is a prerequisite and they were created independently of each other, a merge conflict will need to be resolved.

Reply:
Yes, but this change doesn't need to be on this branch at all. These 5 lines are showing up as a code addition. They have nothing to do with renaming a variable when that variable wasn't originally used here.